pkg/controller/daemon/daemon_controller.go

```diff
@@ -171,13 +192,25 @@ func NewDaemonSetsController(
 				Name: "daemonset-node-updates",
 			},
 		),
+		consistencyStore: consistencyStore,
 	}
 
+	dsGVK := schema.GroupVersionKind{Group: "apps", Version: "v1", Kind: "DaemonSet"}
 	daemonSetInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
+			ds := obj.(*apps.DaemonSet)
+			dsc.consistencyStore.ReadAt(
```

Comment:
I'm not seeing a spot where we record WroteAt for DaemonSet when updating status (added that in https://github.com/liggitt/kubernetes/tree/pocConsistentDs)